gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java

```diff
@@ -1,46 +1,50 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.trino.gateway.ha.security.LbOAuthManager.OidcTokens;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class TestOidcToken {
-  private static final Logger log = LoggerFactory.getLogger(TestOidcToken.class);
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
 
-  @Test
-  public void testParseTokenParamsGracefully() {
-    // This test is to make sure that we simulate the condition where
-    // the OIDC providers send additional parameters in the token
-    // 'to_be_ignored' parameter should not cause any parsing exception
-    // All the other parameters should have the correct values
-    OidcTokens oidcTokens = null;
-    try {
-      ObjectMapper objectMapper = new ObjectMapper();
+public class TestOidcToken
+{
+    private static final Logger log = LoggerFactory.getLogger(TestOidcToken.class);
 
-      String jsonStr = "{\"id_token\" : \"ABC235234\", "
-          + "\"access_token\" : \"AcessABCD123\", "
-          + "\"refresh_token\" : \"RefreshTKN\", "
-          + "\"token_type\" : \"TOKENType\", "
-          + "\"expires_in\" : \"123456\", "
-          + "\"to_be_ignored\" : \"XYX123456\", "
-          + "\"scope\" : \"global\" "
-          + "}";
+    @Test
+    public void testParseTokenParamsGracefully()
+    {
+        // This test is to make sure that we simulate the condition where
+        // the OIDC providers send additional parameters in the token
+        // 'to_be_ignored' parameter should not cause any parsing exception
+        // All the other parameters should have the correct values
+        OidcTokens oidcTokens = null;
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
 
-      oidcTokens = objectMapper.readValue(jsonStr, OidcTokens.class);
-    } catch (JsonProcessingException ex) {
-      log.error(ex.getMessage());
-      assertTrue(false);
-    }
+            String jsonStr = "{\"id_token\" : \"ABC235234\", "
+                    + "\"access_token\" : \"AcessABCD123\", "
+                    + "\"refresh_token\" : \"RefreshTKN\", "
+                    + "\"token_type\" : \"TOKENType\", "
+                    + "\"expires_in\" : \"123456\", "
+                    + "\"to_be_ignored\" : \"XYX123456\", "
+                    + "\"scope\" : \"global\" "
+                    + "}";
 
-    assertTrue(oidcTokens.getIdToken().equals("ABC235234"));
-    assertTrue(oidcTokens.getAccessToken().equals("AcessABCD123"));
-    assertTrue(oidcTokens.getRefreshToken().equals("RefreshTKN"));
-    assertTrue(oidcTokens.getExpiresIn().equals("123456"));
-    assertTrue(oidcTokens.getScope().equals("global"));
-  }
+            oidcTokens = objectMapper.readValue(jsonStr, OidcTokens.class);
+        }
+        catch (JsonProcessingException ex) {
+            log.error(ex.getMessage());
+            fail();
+        }
```

Comment:
Why not remove this catch block?

Reply:
Because I am only doing a reformat and clean up .. not any behavior change .. just to keep it cleaner and simpler .. clean ups should follow for use though